Uzebox/uzebox
# Add back the demos which got removed from the main Makefile, updated …

…their Makefiles so .uze files get built for them

Summary:
 - In a recent git push for "Added the "Akuma" SPI Ram Demo", I noticed that a bunch of other demos were removed from the main Makefile.
 - When added back, those demos did not have .uze files being built for them because of errors in their Makefile.

Changes:
 - Add the removed demos back to the main Makefile
 - Fix each of their Makefiles and the Makefile for SPIRamTesterDemo to use the proper path to packrom (to match all of the other demos)

Testing:
 - make clean
 - make
 - Verify that the roms/ directory contains .hex and .uze files for each demo that had been removed, and SPIRamTesterDemo
 - Verify using cuzebox that each of these .uze files are playable.
